roles/rke2_common/tasks/cis-hardening.yml

```diff
@@ -40,7 +40,7 @@
         remote_src: yes
         mode: 0600
       register: sysctl_operation_tarball
-      when: ansible_facts['os_family'] != 'RedHat' or rke2_binary_tarball_check.stat.exists
+      when: ansible_facts['os_family'] != 'RedHat' or ansible_facts['os_family'] != 'Rocky' or rke2_binary_tarball_check.stat.exists
```

Comment:
Should be an AND operator here. If evaluated for "Rocky" the when statement comes to true because it's !RedHat

Reply:
Fixed